README.rst

```diff
@@ -0,0 +1,154 @@
+tedana: TE Dependent ANAlysis
+=============================
+
+The ``tedana`` package is part of the ME-ICA pipeline, performing TE-dependent
+analysis of multi-echo functional magnetic resonance imaging (fMRI) data.
+``TE``-``de``\pendent ``ana``\lysis (_tedana_) is a Python module for denoising
+multi-echo functional magnetic resonance imaging (fMRI) data.
+
+.. image:: https://img.shields.io/pypi/v/tedana.svg
+   :target: https://pypi.python.org/pypi/tedana/
+   :alt: Latest Version
+
+.. image:: https://img.shields.io/pypi/pyversions/tedana.svg
+   :target: https://pypi.python.org/pypi/tedana/
+   :alt: PyPI - Python Version
+
+.. image:: https://zenodo.org/badge/110845855.svg
+   :target: https://zenodo.org/badge/latestdoi/110845855
+   :alt: DOI
+
+.. image:: https://circleci.com/gh/ME-ICA/tedana.svg?style=svg
```

Comment:
Can we adopt the shield-style badge, here ? Before it was a shield-style in the README and an svg-style badge in the docs, but I think I prefer the shield ! So it would be:
```
.. image:: https://circleci.com/gh/ME-ICA/tedana.svg?style=shield
```

Reply:
👍